.github/workflows/diregapic.yaml

```diff
@@ -0,0 +1,50 @@
+---
+name: DIREGAPIC Updater
+on:
+  schedule:
+    - cron: '0 * * * *'
+  workflow_dispatch:
+
+jobs:
+  regenerate-compute:
+    runs-on: ubuntu-latest
+    container: gcr.io/gapic-images/googleapis:20220222
+    steps:
+    - name: Checkout master
+      uses: actions/checkout@v3
+      with:
+        ref: master
+    - name: Download discovery docs
+      run: |
+        curl https://www.googleapis.com/discovery/v1/apis/compute/v1/rest --output google/cloud/compute/v1/compute.v1.json
+        echo compute_revision=$(grep -oP '"revision":\s*"\d+"' google/cloud/compute/v1/compute.v1.json | grep -oP '\d+') >> $GITHUB_ENV
+    - name: Regenerate API definitions
+      run: |
+        bazel build //google/cloud/compute/v1:compute_gen
+        cp bazel-bin/google/cloud/compute/v1/compute_gen.proto google/cloud/compute/v1/compute.proto
+        bazel build //google/cloud/compute/v1:compute_grpc_service_config_gen
+        cp bazel-bin/google/cloud/compute/v1/compute_grpc_service_config_gen.json google/cloud/compute/v1/compute_grpc_service_config.json
+        bazel build //google/cloud/compute/v1:compute_gapic_gen
+        cp bazel-bin/google/cloud/compute/v1/compute_gapic_gen.yaml google/cloud/compute/v1/compute_gapic.yaml
+        echo api_changes=$(git diff-index --shortstat HEAD) >> $GITHUB_ENV
+    - name: Build GAPIC clients
+      if: contains(env.api_changes, 'files')
+      run: |        
+        bazel build //google/cloud/compute/v1/...
+        bazel build //google/cloud/compute/v1/...
+    - name: Create PR
+      uses: googleapis/code-suggester@v2
+      env:
+        ACCESS_TOKEN: ${{ secrets.YOSHI_CODE_BOT_TOKEN }}
+      with:
+        command: pr
+        upstream_owner: vam-google
+        upstream_repo: googleapis
+        title: 'chore: [DIREGAPIC] Update API definitions'
```

Comment:
This is not good because the bots won't consider the `chore` commit as something that requires release.  Until we are able to generate good release messages (it's under consideration), let's use `feat:` instead to trigger releases.

Reply:
Ok, changed to `feat`